top.mk

```diff
@@ -304,6 +311,11 @@ edk2-clean:
 # Delegate optee targets to optee Makefile
 optee_os/%:
 	${MAKE} -C ${OPTEE_PATH} ${OPTEE_EXTRA} $*
+	if [ $(USE_TEE_BIN) == 'TRUE' ]; then \
+		$(CROSS_COMPILE)objcopy -v -O binary  \
+			$(OPTEE_OUTPUT)/arm-plat-$(OPTEE_PLATFORM)/core/tee.elf \
+			$(OPTEE_OUTPUT)/arm-plat-$(OPTEE_PLATFORM)/core/tee.bin; \
+	fi
```

Comment:
As commented above, I'd rather see tee.bin generation put in a separate target and have that target depend on optee_os/all. That allows this wildcard target to continue working for all optee make targets